S/Sundials/build_tarballs.jl

```diff
@@ -41,26 +42,31 @@ fi
 
 # We attempt to build for all defined platforms
 platforms = supported_platforms()
+platforms = expand_gfortran_versions(platforms)
 
-products = [
-    LibraryProduct("libsundials_sunlinsolspfgmr", :libsundials_sunlinsolspfgmr),
-    LibraryProduct("libsundials_ida", :libsundials_ida),
-    LibraryProduct("libsundials_cvode", :libsundials_cvode),
-    LibraryProduct("libsundials_cvodes", :libsundials_cvodes),
-    LibraryProduct("libsundials_sunmatrixdense", :libsundials_sunmatrixdense),
-    LibraryProduct("libsundials_sunlinsolspbcgs", :libsundials_sunlinsolspbcgs),
-    LibraryProduct("libsundials_idas", :libsundials_idas),
-    LibraryProduct("libsundials_nvecserial", :libsundials_nvecserial),
-    LibraryProduct("libsundials_sunlinsoldense", :libsundials_sunlinsoldense),
-    LibraryProduct("libsundials_sunlinsolspgmr", :libsundials_sunlinsolspgmr),
-    LibraryProduct("libsundials_sunlinsolpcg", :libsundials_sunlinsolpcg),
-    LibraryProduct("libsundials_sunlinsolsptfqmr", :libsundials_sunlinsolsptfqmr),
-    LibraryProduct("libsundials_sunlinsolklu", :libsundials_sunlinsolklu),
-    LibraryProduct("libsundials_sunmatrixsparse", :libsundials_sunmatrixsparse),
-    LibraryProduct("libsundials_sunlinsolband", :libsundials_sunlinsolband),
-    LibraryProduct("libsundials_sunmatrixband", :libsundials_sunmatrixband),
-    LibraryProduct("libsundials_kinsol", :libsundials_kinsol),
-    LibraryProduct("libsundials_arkode", :libsundials_arkode),
+products = [LibraryProduct("libsundials_sunlinsolspfgmr", :libsundials_sunlinsolspfgmr),
```

Comment:
These are the products I get in my local build for `x86_64-linux-gnu`:
```julia
products = [
    LibraryProduct("libsundials_arkode", :libsundials_arkode),
    LibraryProduct("libsundials_cvode", :libsundials_cvode),
    LibraryProduct("libsundials_cvodes", :libsundials_cvodes),
    LibraryProduct("libsundials_ida", :libsundials_ida),
    LibraryProduct("libsundials_idas", :libsundials_idas),
    LibraryProduct("libsundials_kinsol", :libsundials_kinsol),
    LibraryProduct("libsundials_nvecmanyvector", :libsundials_nvecmanyvector),
    LibraryProduct("libsundials_nvecserial", :libsundials_nvecserial),
    LibraryProduct("libsundials_sunlinsolband", :libsundials_sunlinsolband),
    LibraryProduct("libsundials_sunlinsoldense", :libsundials_sunlinsoldense),
    LibraryProduct("libsundials_sunlinsolklu", :libsundials_sunlinsolklu),
    LibraryProduct("libsundials_sunlinsollapackband", :libsundials_sunlinsollapackband),
    LibraryProduct("libsundials_sunlinsollapackdense", :libsundials_sunlinsollapackdense),
    LibraryProduct("libsundials_sunlinsolpcg", :libsundials_sunlinsolpcg),
    LibraryProduct("libsundials_sunlinsolspbcgs", :libsundials_sunlinsolspbcgs),
    LibraryProduct("libsundials_sunlinsolspfgmr", :libsundials_sunlinsolspfgmr),
    LibraryProduct("libsundials_sunlinsolspgmr", :libsundials_sunlinsolspgmr),
    LibraryProduct("libsundials_sunlinsolsptfqmr", :libsundials_sunlinsolsptfqmr),
    LibraryProduct("libsundials_sunmatrixband", :libsundials_sunmatrixband),
    LibraryProduct("libsundials_sunmatrixdense", :libsundials_sunmatrixdense),
    LibraryProduct("libsundials_sunmatrixsparse", :libsundials_sunmatrixsparse),
    LibraryProduct("libsundials_sunnonlinsolfixedpoint", :libsundials_sunnonlinsolfixedpoint),
    LibraryProduct("libsundials_sunnonlinsolnewton", :libsundials_sunnonlinsolnewton),
]
```
I'd also recommend sorting them alphabetically, it makes it easier to go through them.